package-lock.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file isn't related to the changes, looks like a mistaken addition?

Reply:
deleted the file